packages/components/src/components/hds/form/super-select/multiple/base.ts

```diff
@@ -56,7 +58,19 @@ export default class HdsFormSuperSelectMultipleBase extends Component<HdsFormSup
   }
 
   get resultCountMessage(): string {
-    return this.args.resultCountMessage || `${this.optionsCount} total`;
+    if (typeof this.args.resultCountMessage === 'string') {
+      return this.args.resultCountMessage;
+    }
+
+    return `${this.optionsCount} total`;
+  }
+
+  get resultCountMessageFunction() {
```

Comment:
```suggestion
  get resultCountMessageFunction(): PowerSelectSignature['Args']['resultCountMessage'] | undefined {
```
[Nit] Could add typing for function since all other functions in the file are typed.